Mods/ArchipelagoTriggers.SC2Mod/Base.SC2Data/LibABFE498B.galaxy

```diff
@@ -4794,6 +4794,30 @@ void libABFE498B_gf_AP_Triggers_Zerg_unlockSporeCrawler (int lp_player) {
     TechTreeUnitAllow(lp_player, "AP_SporeCrawler", true);
 }
 
+void libABFE498B_gf_AP_Triggers_Zerg_TyrannozorHealingAdapation (int lp_player) {

```

Comment:
Typo -- "Adapation" instead of "Adaptation"

Reply:
fixed